src/chef-mover/rebar.config

```diff
@@ -1,62 +1,74 @@
-%% -*- mode: erlang;erlang-indent-level: 4;indent-tabs-mode: nil -*-
+%% -*- mode: erlang -*-
+%% -*- tab-width: 4;erlang-indent-level: 4;indent-tabs-mode: nil -*-
 %% ex: ts=4 sw=4 ft=erlang et
 
 %% Plugin usage
 {cover_enabled, false}.
 
 {deps, [
-        {lager, ".*",
-         {git, "https://github.com/basho/lager", {branch, "master"}}},
-        {opscoderl_wm, ".*",
-         {git, "https://github.com/opscode/opscoderl_wm", {branch, "master"}}},
-        {chef_authn, ".*",
-         {git, "https://github.com/chef/chef_authn", {branch, "master"}}},
-        {moser, ".*",
-         {git, "https://github.com/opscode/moser", {branch, "master"}}},
-        {jiffy, ".*",
-         {git, "https://github.com/davisp/jiffy", {branch, "master"}}},
-        {chef_reindex, ".*",
-         {git, "https://github.com/opscode/chef_reindex", {branch, "master"}}},
-        {ibrowse, ".*",
-         {git, "https://github.com/opscode/ibrowse", {tag, "v4.0.1.1"}}},
-        {eper, ".*",
-         {git, "https://github.com/massemanet/eper", {branch, "master"}}},
-        {folsom, ".*",
-         {git,"https://github.com/boundary/folsom", {tag, "0.7.4"}}},
-        {eredis, ".*",
-         {git, "https://github.com/wooga/eredis", {tag, "master"}}},
-        {stats_hero, ".*",
-         {git, "https://github.com/opscode/stats_hero", {branch, "master"}}},
-        {bcrypt, ".*",
-         {git, "https://github.com/opscode/erlang-bcrypt", {branch, "master"}}},
-        {sqerl,".*",
-         {git,"https://github.com/chef/sqerl", {branch, "master"}}},
-        {epgsql,".*",
-         {git,"https://github.com/chef/epgsql-1", {branch, "master"}}},
-        {chef_secrets, ".*",
-         {git, "https://github.com/chef/chef_secrets", {branch, "master"}}},
+    %% lager has to come first since we use its parse transform
+    {lager, ".*",
+        {git, "https://github.com/basho/lager.git",                     {branch, "master"}}},
+    {bcrypt, ".*",
+        {git, "https://github.com/smarkets/erlang-bcrypt.git",          {branch, "master"}}},
+    {chef_authn, ".*",
+        {git, "https://github.com/chef/chef_authn.git",                 {branch, "master"}}},
+    {chef_reindex, ".*",
+        {git, "https://github.com/opscode/chef_reindex.git",            {branch, "master"}}},
+    {chef_secrets, ".*",
+        {git, "https://github.com/chef/chef_secrets.git",               {branch, "master"}}},
+    {edown, ".*",
+        {git, "https://github.com/uwiger/edown.git",                    {branch, "master"}}},
+    {ej, ".*",
+        {git, "https://github.com/seth/ej.git",                         {branch, "master"}}},
+    {eper, ".*",
+        {git, "https://github.com/massemanet/eper.git",                 {branch, "master"}}},
+    {epgsql,".*",
+        {git, "https://github.com/chef/epgsql-1.git",                   {branch, "master"}}},
+    {eredis, ".*",
+        {git, "https://github.com/wooga/eredis.git",                    {tag, "master"}}},
```

Comment:
Also {tag, "master"} => {branch, "master"}

Reply:
was {tag, "master"} in master branch.  now changed to {branch, master}.